Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -329,6 +329,7 @@ void JitArm64::frspx(UGeckoInstruction inst)
 {
 	INSTRUCTION_START
 	JITDISABLE(bJITFloatingPointOff);
+	FALLBACK_IF(inst.Rc);
```

Comment:
Seems like I'm wrong.
flacs> frspx *does* depend on the Rc bit (that's what the 'x' stands for, remember?)

But maybe we should also fix the interpreter itself here. Maybe also in https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/PowerPC/Jit64/Jit_FloatingPoint.cpp#L611 ?